documentation/how-to/search.en.md

```diff
@@ -18,7 +18,7 @@ open expand the search bar.
 
 The value entered in the search bar will be used to find features with
 matching attribute values. A minimum of three characters is required to
-start the search.
+start the search. Additionally, you have the option to employ the code scan functionality for this purpose.
```

Comment:
I believe here we can elaborate HOW it works and we can suggest encoding your feature UUIDs, so we can find them right away.

Reply:
Done!